Do-It/Controller/DoItTableViewController.swift

```diff
@@ -39,36 +42,55 @@ class DoItTableViewController: UIViewController, UITableViewDelegate, UITableVie
                           name: "DIFF",
                           priority: DoItPriority.low,
                           kind: DoItKind.homework))
+        filteredDoIts = doIts
 
         tableView.delegate = self
         tableView.dataSource = self
 
+        navigationEditItem.rightBarButtonItem = UIBarButtonItem(title: "Edit",
+                                                                style: UIBarButtonItem.Style.plain,
+                                                                target: self,
+                                      action: #selector(editButtonPressed))
+
+        searchBar  = UISearchBar(frame: CGRect(origin: .zero,
+                                               size: CGSize(width: UIScreen.main.bounds.width,
+                                                            height: 44)))
+        searchBar.delegate = self
     }
```

Comment:
All of this in `viewWillAppear` can be in `viewDidLoad`, since it's only necessary to do once.